docs/source/workflow_manager.rst

```diff
@@ -0,0 +1,308 @@
+The Workflow Manager
+====================
+
+The workflow manager (WM) allows the user to easily construct a graph of functions,
+optimize this graph and view the results graphically. A number of terms are specific
+to BDSS/WM or otherwise have a specific meaning within BDSS/WM.
+
+**Workflow**
+    The directed graph of functions that models a process.
+
+**Data source**
+    A node (function) in the graph (workflow).
+
+**Data value**
+    An input or output of a node (data source).
+
+**Parameter**
+    An input of the graph (workflow) and how that input should be treated (as a numerical or categorical variable, etc: see below).
+
+**Key performance indicator (KPI)**
+    An output (criterion or objective) of the graph (workflow).
+
+**MCO**.
+    Multi-criterion optimizer.
+
+Views
+-----
+
+The WM has two "views":
```

Comment:
Cool, the annotations are numbered, so we could also probably steal the table sections from that document as a brief explanation too

Reply:
This is more design documentation rather than a step-by-step how-to tutorial.  I don't think the screenshots and/or the tables, alone would help in this context.